system-contracts/contracts/PasskeyBinder.sol

```diff
@@ -65,50 +59,54 @@ contract PasskeyBinder is IPasskeyBinder {
 
         if (_credentialIdHash == bytes32(0)) revert InvalidCredentialIdHash();
 
-        AuthorizedKey storage authorizedKey = authorizedKeys[_credentialIdHash];
-
-        if (authorizedKey.owner != address(0)) revert KeyAlreadyExists(_credentialIdHash);
+        address authorizedKeyOwner = authorizedKeys[_credentialIdHash].owner;
+        if (authorizedKeyOwner == address(0)) {
+            authorizedKeys[_credentialIdHash] = AuthorizedKey(sender, P256PublicKey(_x, _y));
```

Comment:
how to prevent fake (x, y) attack?

Reply:
(x, y) is obtained from the Passkey generated by the device. When a fake (x, y) is bound, the signature check fails.

Reply:
Front-Running, there is a possibility of contaminating other users' passwords. But it has been fixed.